### ISSUE TITLE
[3.0.0-rc1] System.ArgumentException: An item with the same key has already been added. Key: .wasm

### ISSUE THREAD

**Issue by tebeco** [dotnet/try#445]
#### Describe the bug
Can't start `dotnet try` :
System.ArgumentException: An item with the same key has already been added. Key: .wasm

#### Did this error occur while using `dotnet try` or online?
- [x] `dotnet-try`

#### What kind of error was it?
- [x] Other

#### Please complete the following:
 - OS
    - [x] Windows 10

Hello,

As @jonsequitur pointed [here](https://github.com/dotnet/try/issues/166) ~7month ago, this issue has been identified [here](https://github.com/aspnet/AspNetCore/issues/5666) 
and probably fixed [here](https://github.com/aspnet/AspNetCore/pull/6731/files)

Is there a branch, a version, anything to make `dotnet try` run ?

i tried all 3 version on `nuget.org` as global tool
i also tried the latest version installed with
```
> dotnet install tool dotnet-try --tool-path ./.try
> ./.try/dotnet-try.exe .
```

I tried :
* with empty folder
* on this repo sample
* with the `dotnet/try-sample` repo
* with `Options 2` from the docs

I'm not sure what i missed, but i guess something obvious but i can't get it ^^

**Comment by jonsequitur** [comment]
Could you run `dotnet --info` and provide the output?

**Comment by tebeco** [comment]
On a random folder :

```
> d --info                                                                                                                                                                                                                                                                                            .NET Core SDK (reflecting any global.json):                                                                                                                                                                                                                                                                             Version:   3.0.100-preview9-014004                                                                                                                                                                                                                                                                                     Commit:    8e7ef240a5                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Runtime Environment:                                                                                                                                                                                                                                                                                                    OS Name:     Windows                                                                                                                                                                                                                                                                                                   OS Version:  10.0.18980                                                                                                                                                                                                                                                                                                OS Platform: Windows                                                                                                                                                                                                                                                                                                   RID:         win10-x64                                                                                                                                                                                                                                                                                                 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview9-014004\                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Host (useful for support):                                                                                                                                                                                                                                                                                               Version: 3.0.0-preview9-19423-09                                                                                                                                                                                                                                                                                       Commit:  2be172345a                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         .NET Core SDKs installed:                                                                                                                                                                                                                                                                                                2.2.401 [C:\Program Files\dotnet\sdk]                                                                                                                                                                                                                                                                                  3.0.100-preview9-014004 [C:\Program Files\dotnet\sdk]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       .NET Core runtimes installed:                                                                                                                                                                                                                                                                                            Microsoft.AspNetCore.All 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]                                                                                                                                                                                                                               Microsoft.AspNetCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]                                                                                                                                                                                                                               Microsoft.AspNetCore.App 3.0.0-preview7.19365.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]                                                                                                                                                                                                              Microsoft.AspNetCore.App 3.0.0-preview9.19424.4 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]                                                                                                                                                                                                              Microsoft.NETCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]                                                                                                                                                                                                                                     Microsoft.NETCore.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]                                                                                                                                                                                                                   Microsoft.NETCore.App 3.0.0-preview9-19423-09 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]                                                                                                                                                                                                                   Microsoft.WindowsDesktop.App 3.0.0-preview7-27912-14 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]                                                                                                                                                                                                     Microsoft.WindowsDesktop.App 3.0.0-preview9-19423-09 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          To install additional .NET Core runtimes or SDKs:                                                                                                                                                                                                                                                                        https://aka.ms/dotnet-download                                                                                                                                                                                                                                                                                       ```

inside the `dotnet-try` repo, same.

then i tried to update the repo and run `dotnet/arcade` on it to use the "shipped" sdk ...
```
PS C:\workspace\github\dotnet\try> git reset --hard
HEAD is now at 7c88dc74 Merge pull request #425 from colombod/feature/notebook_examples
PS C:\workspace\github\dotnet\try> git clean -fdx
Removing artifacts/
PS C:\workspace\github\dotnet\try> git clean -fdx
PS C:\workspace\github\dotnet\try> git reset --hard
HEAD is now at 7c88dc74 Merge pull request #425 from colombod/feature/notebook_examples
PS C:\workspace\github\dotnet\try> git pull
Already up to date.
PS C:\workspace\github\dotnet\try> .\build.cmd
(NETCORE_ENGINEERING_TELEMETRY=InitializeToolset) Toolset version 1.0.0-beta.19461.7 has not been restored.
```

**Comment by tebeco** [comment]
(same on `rc1`)

**Comment by jsolana** [comment]
Hi guys, i ve the same error following both options described in https://github.com/dotnet/try#getting-started  

My dotnet --info 

```text
SDK de .NET Core (reflejando cualquier global.json):
 Version:   3.0.100
 Commit:    04339c3a26

Entorno de tiempo de ejecución:
 OS Name:     ubuntu
 OS Version:  19.04
 OS Platform: Linux
 RID:         ubuntu.19.04-x64
 Base Path:   /usr/share/dotnet/sdk/3.0.100/

Host (useful for support):
  Version: 3.0.0
  Commit:  95a0a61858

.NET Core SDKs installed:
  2.2.402 [/usr/share/dotnet/sdk]
  3.0.100 [/usr/share/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.0.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.7 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by tebeco** [comment]
@jonsequitur 
i haven't retested since a long time ago as it was kinda blocking
i suppose it's probably been fixed since 3.0 and 3.1 were released
should I close this issue ?
